ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -249,4 +259,69 @@ public void testDeprecation() {
 
     pipeline.run();
   }
+
+  @Test
+  public void testPingSplitting() throws IOException {
+    String schemasLocation = TestConstant.SCHEMAS_LOCATION;
+    SchemaStoreSingletonFactory.clearSingletonsForTests();
```

Comment:
because this `schemasLocation` is different from the one used in all the other tests in this file, and the singleton factory causes `schemasLocation` to be ignored if schemas have already been loaded.